locales/en/smart-events.json

```
@@ -113,7 +114,8 @@
     "selectRegion": "Select region",
     "smartEventInstance": "Smart Events Instance",
     "smartEventInstances": "Smart Event Instances",
-    "viewInformation": "View instance information"
+    "viewInformation": "View instance information",
+    "quotaExceededErrorDescription": "To deploy a new instance, delete an existing one first"
```

Nitpicking, missing dot at the end.

```suggestion
    "quotaExceededErrorDescription": "To deploy a new instance, delete an existing one first."
```